requirements.txt

```diff
@@ -1,3 +1,3 @@
-anta[cli]==1.2.0
+anta[cli]==1.5.0
 passlib
-pyavd[ansible]==5.2.2
\ No newline at end of file
+pyavd[ansible]==5.7.0
```

Comment:
Let's update to 5.7.1, and I recommend we do a pip freeze and pin all dependencies to a specific version
```suggestion
pyavd[ansible]==5.7.1
```